### DISCUSSION TITLE
Why is it that when a transaction is cancelled, the product stock in that transaction does not return to its original state?

### DISCUSSION THREAD

**Discussion by craftalpian** [medusajs/medusa#5376] [Q&A]
I want that when someone makes a transaction on my website, and does not make payment within a certain time, our operator will cancel the transaction and hope that the product stock in that variant will return to normal. But in fact it's not, why does that happen? Isn't that a basic function when a transaction is canceled then the product stock returns to its original value? Or is this feature in the latest version?

My medusa version: 1.13.0

**Comment by craftalpian** [comment]
Any answer? @olivermrbl @shahednasser

**Reply by Julian-Dumitrascu** [reply]
Does this team want to respond within a fortnight? I want to know the related view of the person who decides on such things.

**Comment by srindom** [comment]
This sounds like a bug - can you open an issue with steps to reproduce :)
